src/Aspire.Hosting.GitHub.Models/GitHubModel.Obsolete.cs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.GitHub;
+
+// This file contains obsolete elements kept for backward compatibility.
+
+public partial class GitHubModel
+{
+    public static partial class Core42
+    {
+        /// <inheritdoc cref="Jais30bChat"/>
+        [Obsolete("Use Jais30bChat instead.")]
+        public static readonly GitHubModel JAIS30bChat = new() { Id = "core42/jais-30b-chat" };
```

Comment:
Add `[EditorBrowsable(EditorBrowsableState.Never)]` to these fields?

Reply:
Good idea, will also do it for foundry.